cfgov/jinja2/v1/_includes/atoms/submit-button.html

```diff
@@ -1,5 +1,9 @@
+{% macro render(value) -%}
+
 <div class="o-form-actions">
     <input class="btn btn__super form-actions_item"
            type="submit"
            value="{{ value.button_text }}">
 </div>
+
+{%- endmacro %}
```

Comment:
Is this atom being called anywhere else that will break due to it being put into a macro?